pillar/base/haproxy.sls

```diff
@@ -67,6 +67,29 @@ haproxy:
       verify_host: planet.psf.io
       check: "HEAD / HTTP/1.1\\r\\nHost:\\ planet.psf.io"
 
+    {% for (port, service) in [(25, "smtp"), (587, "smtps"), (465, "submission")] %}
```

Comment:
I think this needs to be in the salt state for `bugs`, not the haproxy configuration file.

probably around [here](https://github.com/python/psf-salt/blob/448b8a7c0007bf3ffaf7098b3363956401e82929/salt/bugs/init.sls#L212).